src/UserContainer.java

```diff
@@ -1,19 +1,23 @@
-import java.util.HashMap;
-import java.util.Map;
+import java.util.ArrayList;
+import java.util.List;
 
 public class UserContainer {
-   private final static Map<String, String> users = new HashMap<String, String>();
 
-    public static Map<String, String> getUsers() {
+    private final static List<User> users = new ArrayList();
+
+    public final static List<User> getUsers() {
         return users;
     }
 
     public static boolean isUserNameAndPasswordExists(String userName, String password) {
-        boolean userExists = false;
-
-        if (users.containsKey(userName) && users.containsValue(password)) {
-            userExists = true;
+        boolean existUser = false;
+        for (int i = 0; i < users.size(); i++) {
+            if (users.get(i).getUsername().equals(userName)) {
+                if (users.get(i).getPassword().equals(password)) {
```

Comment:
maybe it is easier to find user
and then get username and get passowrd from this user?

now you get user by index every time